crates/host/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "wasmcloud-host"
```

Comment:
This is a good name for this crate, we should consider renaming the wasmcloud dashboard from `wasmcloud_host` to avoid confusion 🤔

Reply:
@brooksmtownsend could you file a ticket/issue regarding that in https://github.com/wasmCloud/wasmcloud-otp ?